### PR TITLE
Add client_user_id field to POST /item/add_token/create

### PR DESCRIPTION
**Pull request by nathankot**
(Tests will only pass when we introduce the field on our end)

### CODE REVIEW

File: lib/PlaidClient.js [hunk 1]
```diff
@@ -90,11 +90,14 @@ Client.prototype.createItemAddToken =
       options = {};
     }
     const {
+      client_user_id,
       user_identity,
     } = options || {};
     return this._authenticatedRequest({
       path: '/item/add_token/create',
-      body: helpers.processUserFields(user_identity),
+      body: R.merge(
```
Comment (otherchen) [inline lib/PlaidClient.js:98]: What do you think about merging `client_user_id` into `user_identity` since we do something similar in the API?

Comment (nathankot) [reply lib/PlaidClient.js]: ⚡ 689caa5

File: lib/_helpers.js [hunk 2]
```diff
@@ -7,21 +7,31 @@ const R = require('ramda');
...
-      R.merge(user, processField(f, user_identity[f] || {}))
-    ), {});
+  const userIdentityFields = [
+    'email_address',
+    'phone_number',
+    'legal_name',
+  ];
 
-  if (Object.keys(user).length === 0) {
+  const userForAPI = R.merge(
```
Comment (otherchen) [inline lib/_helpers.js:21]: 👍

File: lib/_helpers.js [hunk 3]
```diff
@@ -7,21 +7,31 @@ const R = require('ramda');
...
+    user.client_user_id == null ? {} :
+    { client_user_id: user.client_user_id },
+    userIdentityFields
+      .reduce((userForAPI, f) => (
```
Comment (otherchen) [inline lib/_helpers.js:25]: 🙈 so hard to read lol

**Pull request merged**